packages/user/src/User/Login.purs

```diff
@@ -474,101 +415,6 @@ renderMerge self@{ props } mergeInfo =
-googleLogin self =
-  if not $ Set.member Google self.props.disableSocialLogins
-    then
-      someLoginButton
-        { className: "login--some-button login--some-button-google"
-        , description: "Logga in med Google"
-        , onClick: Google.loadGapi { onSuccess: onGoogleLogin, onFailure: onGoogleFailure }
-        }
-    else mempty
-  where
-    onGoogleLogin :: Google.AuthResponse -> Effect Unit
-    onGoogleLogin { accessToken, email: (Google.Email email) } = self.props.onLogin do
-      failOnEmailMismatch self email
-      -- setting the email in the state to eventually have it in the merge view
-      liftEffect $ self.setState _ { formEmail = Just email }
-      loginWithRetry (logUserIn email accessToken) handleSuccess handleError handleFinish
-
-    logUserIn email accessToken = User.someAuth Nothing self.state.merge (User.Email email) (User.Token accessToken) User.GooglePlus
-    handleSuccess user = liftEffect $ Tracking.login (Just user.cusno) "google login" "success"
-    handleError _err = liftEffect $ Tracking.login Nothing "google login" "error"
-    handleFinish user = do
-      finalizeSomeAuth self user
-      liftEffect $ self.props.onUserFetch user
-
-    -- | Handles Google login errors. The matched cases are:
-    -- | 1) Error "idpiframe_initialization_failed".
-    -- |    Google auth initialization failure (probably due to not allowing 3rd party cookies).
-    -- | 2) Error "popup_closed_by_user".
-    -- |    Ignore this, as it's not an error we want to catch.
-    -- | 3) Every other auth failure.
-    -- |    TODO: At least some of these should be displayed to the user.
-    onGoogleFailure :: Google.Error -> Effect Unit
-    onGoogleFailure { error: "idpiframe_initialization_failed", details }
-      -- In addition to failing because of third party cookies, the initialization may
-      -- fail e.g. for the reason, that the domain is not allowed to perform any google actions.
-      -- This case is just to let the developers know what's going on.
-      | not $ isGoogleDomainAllowed details
-      = do
-        Log.error $ "You need to allow this origin, dummy: " <> details
-        self.setState _ { errors { login = Just LoginGoogleAuthInitErrorOrigin } }
-      | otherwise = self.setState _ { errors { login = Just LoginGoogleAuthInitError } }
-    onGoogleFailure { error: "popup_closed_by_user" } = pure unit
-    onGoogleFailure _ = Log.error "Google login failed."
-
-    isGoogleDomainAllowed = not <<< contains (Pattern "Not a valid origin for the client")
-
--- | The Facebook login is disabled. Instead of the button, we just
---   show a text explaining it's gone
-facebookLogin :: Self -> JSX
-facebookLogin self =
-  guard (not $ Set.member Facebook self.props.disableSocialLogins)
-    DOM.div
-      { className: "login--some-disabled-note"
-      , children: [ DOM.text "Möjligheten att logga in med Facebook har tagits bort. Du kan logga in med den e-postadress som är kopplad till din Facebook, "
```

Comment:
We need to add this text (and also something about google login) to the component somewhere